.github/workflows/dotnet.yml

```
@@ -0,0 +1,21 @@
+      matrix:
+        dotnet-version: [ '9.0.x']
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup dotnet ${{ matrix.dotnet-version }}
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: ${{ matrix.dotnet-version }}
+      # You can test your matrix by printing the current dotnet version
+      - name: Display dotnet version
+        run: dotnet --version
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)